source/new.rst

```diff
@@ -680,9 +665,14 @@ Notes on the verified group protocol
   are still impossible.
 
   A malicious verified contact may inject MITM keys.
-  We note, however,
-  that such an infiltrator (say Bob when adding Carol as a new member),
-  will have to sign the message containing the gossip fake keys.
+  Say Bob when adding Carol as a new member,
+  sends a prepared MITM key.
+  We refer to this as a Bob in the middle attack
+  to illustrate that a peer is involved in the attack.
+
+  We note,
+  that Bob, will have to sign the message
+  containing the gossip fake keys.
   If Carol performs a verification with Alice,
   she can use Bob's signature to prove
   that Bob gossiped the wrong key for Alice.
```

Comment:
reference or mention "keyhistory" here -- it's a bit impilcit in "if carl performs a verification with alice".